### PR TITLE
port K8s rolling upgrade guidance back to v20.2

### PR DESCRIPTION
**Pull request by taroface**
Updates in #8615 (specifically for partitions) were lost in v20.2 when adding K8s Operator steps. This commit brings them back, but at the slight expense of imposing a less efficient structure on the doc. This was necessary in order to maintain distinct instructions for Operator/Helm/configs without losing enumeration.

### CODE REVIEW

**Comment by taroface** [comment]
@jseldess Would you mind giving this a quick docs review? All content in this PR has previously been approved, and the only change is to how the "upgrades" include is formatted. I think this is a sign that it's time to break up these operational docs.

**Review by jseldess — APPROVED** [review]
LGTM, but I didn't have the chance to actually test out these docs changes. If you have, we're good.

File: _includes/v20.2/orchestration/kubernetes-upgrade-cluster.md [hunk 1]
```diff
@@ -82,54 +173,255 @@ The corresponding process on Kubernetes is a [staged update](https://kubernetes.
...
+    ~~~
+
+    ~~~
+    cockroachdb-0   cockroachdb/cockroach:{{page.release_info.version}}
+    cockroachdb-1   cockroachdb/cockroach:{{page.release_info.version}}
+    cockroachdb-2   cockroachdb/cockroach:{{page.release_info.version}}
+    ...
+    ~~~
+
+    You can also check the CockroachDB version of each node in the [Admin UI](admin-ui-cluster-overview-page.html#node-details).
```
Comment (jseldess) [inline _includes/v20.2/orchestration/kubernetes-upgrade-cluster.md:301]: Admin UI > DB Console

File: _includes/v20.2/orchestration/kubernetes-upgrade-cluster.md [hunk 2]
```diff
@@ -190,45 +459,85 @@ The corresponding process on Kubernetes is a [staged update](https://kubernetes.
...
     ~~~
     my-release-cockroachdb-0    cockroachdb/cockroach:{{page.release_info.version}}
     my-release-cockroachdb-1    cockroachdb/cockroach:{{page.release_info.version}}
     my-release-cockroachdb-2    cockroachdb/cockroach:{{page.release_info.version}}
     ...
     ~~~
-    </section>
 
-    You can also check the CockroachDB version of each node in the [DB Console](ui-cluster-overview-page.html#node-details).
+    You can also check the CockroachDB version of each node in the [Admin UI](admin-ui-cluster-overview-page.html#node-details).
```
Comment (jseldess) [inline _includes/v20.2/orchestration/kubernetes-upgrade-cluster.md:539]: Admin UI > DB Console

**Review by taroface — COMMENTED** [review]
Yep, tested them with the original PR!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8910#-:-MMHceg09y8ErNY9j1aS:bi3ci4x)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @jseldess)

___
*[_includes/v20.2/orchestration/kubernetes-upgrade-cluster.md, line 301 at r1](https://reviewable.io/reviews/cockroachdb/docs/8910#-MLyUZmnXwp9Q7UoFk8c-r1-301:-MMHchQC-_kFHn8Srezt:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/7baa22c0ca7574481584e5a677b01470bad54e1a/_includes/v20.2/orchestration/kubernetes-upgrade-cluster.md#L301)):*
<details><summary><i>Previously, jseldess (Jesse Seldess) wrote…</i></summary><blockquote>

Admin UI > DB Console
</blockquote></details>

Done.
___
*[_includes/v20.2/orchestration/kubernetes-upgrade-cluster.md, line 539 at r1](https://reviewable.io/reviews/cockroachdb/docs/8910#-MLyUZmnXwp9Q7UoFk8c-r1-539:-MMHchkUCqLiX3F4gGVz:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/7baa22c0ca7574481584e5a677b01470bad54e1a/_includes/v20.2/orchestration/kubernetes-upgrade-cluster.md#L539)):*
<details><summary><i>Previously, jseldess (Jesse Seldess) wrote…</i></summary><blockquote>

Admin UI > DB Console
</blockquote></details>

Done.

**Pull request merged**